packages/terser/src/index.ts

```diff
@@ -1,24 +1,8 @@
-import { NormalizedOutputOptions, RenderedChunk } from 'rollup';
-import { minify, MinifyOptions } from 'terser';
+import { runWorker } from './worker';
+import terser from './module';
 
-export default function terser(options?: MinifyOptions) {
-  return {
-    name: 'terser',
+Promise.resolve().then(() => runWorker());
```

Comment:
Is this line needed?

Reply:
In theory not, but my ide (webstorm) is complaining about it and eslint in some configurations might too, due the fact that the `runWorker` method is async.

Reply:
was speaks against the usage of `Promise.resolve()` ? 
Are there any impacts or undesired behaviour ?

Reply:
good point :blush: . I removed it :v: